attributes/default.rb

```diff
@@ -70,6 +70,8 @@
 # Leave key value to nil to use agent6 default for that endpoint.
 # Supported keys: "series", "events", "service checks"
 default['datadog']['use_v2_api'] = {}
+# set path to dogstatsd unix socket - https://docs.datadoghq.com/developers/dogstatsd/unix_socket/
+default['datadog']['dogstatsd_socket'] = false
```

Comment:
Can you leave the default value to `nil` here? In general for agent options we prefer letting the Agent determine what the correct default instead of the cookbook.